cpplint/cpplint_unittest.py

```diff
@@ -3071,7 +3072,8 @@ def DoTest(self, raw_bytes, has_invalid_utf8):
       error_collector = ErrorCollector(self.assert_)
       cpplint.ProcessFileData(
           'foo.cc', 'cc',
-          unicode(raw_bytes, 'utf8', 'replace').split('\n'),
+          six.ensure_text(raw_bytes, encoding='utf8',
+            errors='replace').split('\n'),
```

Comment:
indent is off -- it should align with ( above

that said, you don't need six.  just write `raw_bytes.decode('utf-8', 'replace')`.

Reply:
OK.